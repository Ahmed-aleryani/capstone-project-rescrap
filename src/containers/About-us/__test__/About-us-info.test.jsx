import React from 'react';
import renderer from 'react-test-renderer';
import AboutUsInfo from '../About-us-info';

it("matches the snapshot", () =>{
    const tree = renderer.create(<AboutUsInfo/>).toJSON();
    expect(tree).toMatchSnapshot();
})