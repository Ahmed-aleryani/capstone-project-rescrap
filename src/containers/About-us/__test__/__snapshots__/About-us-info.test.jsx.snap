// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className=""
>
  <h2
    className="text-2xl thintext-5xl text-center text-green-100 m-4 "
  >
    translation:pages.aboutus.subtitle
  </h2>
  <div
    className="text-base flex flex-col md:flex-row lg:flex-row xl:flex-row mx-8 mb-16"
  >
    <p
      className="p-8 text-justify"
    >
      translation:pages.aboutus.paragraph1
    </p>
    <p
      className="p-8 text-justify"
    >
      translation:pages.aboutus.paragraph2
    </p>
  </div>
</div>
`;
